packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -485,16 +487,19 @@ contract OPContractsManager is ISemver, Initializable {
                 ISystemConfig.Addresses memory opChainAddrs
             ) = defaultSystemConfigParams(_selector, _input, _output);
 
+            address batchInboxAddress = chainIdToBatchInboxAddress(_input.l2ChainId);
             return abi.encodeWithSelector(
                 _selector,
                 _input.roles.systemConfigOwner,
                 _input.basefeeScalar,
                 _input.blobBasefeeScalar,
                 bytes32(uint256(uint160(_input.roles.batcher))), // batcherHash
                 _input.gasLimit,
+                _input.eip1559Denominator,
+                _input.eip1559Elasticity,
```

Comment:
Can you make these changes via the inheritance pattern described in https://github.com/ethereum-optimism/design-docs/blob/main/smart-contract-feature-development.md, similar to what e.g. `OptimismPortalInterop` and `OptimismPortalInterop` do? We do not yet have the final OPCM versions deployed on testnet or mainnet, and this change is not compatible with the `op-contracts/v1.6.0` release the initial OPCM deploy needs to support

Reply:
Oh actually - do we need to do this? There's already a branch there for backwards compatibility with `op-contracts/v1.6.0`, unsure of if that covers your ask here.

Reply:
Ah yep you're right! Sorry, didn't realize we were within the `else` clause here since github collapsed the diff